test/testcases/hash_bytes.ts

```diff
@@ -0,0 +1,99 @@
+import * as path from 'path';
```

Comment:
this test is not run, and seems incorrect? (correct version is in test/testcases/hash_txdata.ts).

seems we should remove this file?